cogboard-app/src/main/kotlin/com/cognifide/cogboard/config/service/EndpointsService.kt

```diff
@@ -1,24 +1,19 @@
 package com.cognifide.cogboard.config.service
 
-import com.cognifide.cogboard.config.ConfigType
 import com.cognifide.cogboard.config.EndpointLoader
 import com.cognifide.cogboard.config.EndpointsConfig
 import com.cognifide.cogboard.config.EndpointsConfig.Companion.ENDPOINTS_ARRAY
 import com.cognifide.cogboard.config.EndpointsConfig.Companion.ENDPOINT_ID_PREFIX
 import com.cognifide.cogboard.config.EndpointsConfig.Companion.ENDPOINT_ID_PROP
 import com.cognifide.cogboard.config.EndpointsConfig.Companion.ENDPOINT_LABEL_PROP
 import com.cognifide.cogboard.storage.Storage
-import com.cognifide.cogboard.storage.VolumeStorage
 import com.cognifide.cogboard.config.utils.JsonUtils.getObjectPositionById
 import com.cognifide.cogboard.config.utils.JsonUtils.putIfNotExist
 import com.cognifide.cogboard.config.validation.endpoints.EndpointsValidator
-import io.vertx.core.Vertx
 import io.vertx.core.json.JsonArray
 import io.vertx.core.json.JsonObject
 
-class EndpointsService(private val config: JsonObject, vertx: Vertx) {
-
-    private var storage: Storage = VolumeStorage(ConfigType.ENDPOINTS, vertx)
+class EndpointsService(private var storage: Storage, private val config: JsonObject = storage.loadConfig()) {
```

Comment:
Whitespace: in `WidgetRuntimeService` you add newlines for parameters, and here you don't. I'd add it here as well, just for clarity.